hydroflow_lang/src/graph/ops/dest_sink.rs

```diff
@@ -15,7 +15,7 @@ use quote::quote_spanned;
 /// Note this operator must be used within a Tokio runtime.
 ///
 /// ```rustbook
-/// # #[tokio::main(flavor = "current_thread")]
+/// # #[hydroflow::hydroflow_main]
```

Comment:
Would be nice to rename or re-export these so they are `hydroflow::main` and `hydroflow::test` instead of the doubled up version. Might also shrink the snapshot diff